test/kfp-functional-test/requirements.txt

```diff
@@ -85,7 +85,7 @@ requests==2.32.2
     #   requests-toolbelt
 requests-oauthlib==2.0.0
     # via kubernetes
-requests-toolbelt==0.10.1
+requests-toolbelt==1.0.0
```

Comment:
Did you execute `pip-compile requirements.in` to genereta the requirements.txt?

Reply:
I tried, but now the requests-toolbelt version is back to 0.10.1 here

Reply:
Double-check the dependency tree and see what library is requesting `requests-toolbelt` be with `0.10.1`

Reply:
it's kfp(==2.9.0), but I'm not sure how to fix that without releasing and versioning the other change.

Reply:
cc @chensun what we should do in this case?

Reply:
Try the following:
```
pip install sdk/python/
pip-compile requirements.in
```
In this case, we won't need to have a version before your change.

Please let me know if this will work.

Reply:
it doesn't work, I'm afraid, `pip-compile` still looks to PyPI it seems. If in requirements.in I replace `kfp` by `../../sdk/python` it does work, but I'm guessing that's not a good solution.

Reply:
@chensun any idea?

Reply:
@chensun could you take a look?

Reply:
@diegolovison @SimonDR-Boltzmann I believe this is because it's installing kfp from pypi repo, which still uses the old library version. What you could try is temporarily test adding the `git+ssh` entry in `requirements.in` pointing to your branch and see if `pip-compile requirements.in` upgrades to 1.0.

cc @gregsheremeta for awareness.

Reply:
It looks like the tests succeed when I do this  (see https://github.com/kubeflow/pipelines/pull/10762/files#diff-bd584e00f9357e94370a0ebb8f92990519014c49c20206352a17c430f3b79327R2).
I did have to specify `requests-toolbelt==1.0.0` in the requirements.in before pip-compile used the new version, but there were no conflicts.

Shall I now remove the changes in `test/kfp-functional-test`, for final review?